audioled/serverconfiguration.py

```diff
@@ -173,6 +173,10 @@ def createOutputDevice(self):
             device = devices.FadeCandy(
                 self.getConfiguration(CONFIG_NUM_PIXELS), self.getConfiguration(CONFIG_NUM_ROWS),
                 self.getConfiguration(CONFIG_DEVICE_CANDY_SERVER))
+        elif self.getConfiguration(CONFIG_DEVICE) == devices.DotStar.__name__:
+            device = devices.DotStar(
+                self.getConfiguration(CONFIG_NUM_PIXELS), self.getConfiguration(CONFIG_NUM_ROWS),
+                self.getConfiguration(CONFIG_DEVICE_CANDY_SERVER))
```

Comment:
This violates DRY. I suggest using getattr to instantiante the class dynamically.

```
   def createOutputDevice(self):
            device = getattr(devices, self.getConfiguration(CONFIG_DEVICE))(
                self.getConfiguration(CONFIG_NUM_PIXELS), self.getConfiguration(CONFIG_NUM_ROWS))
```